### PR TITLE
BXMSDOC-7465 Consolidate Dashbuilder content

### PR DESCRIPTION
**Pull request by emmurphy1**
[BXMSDOC-7465](https://issues.redhat.com/browse/BXMSDOC-7465)

### CODE REVIEW

File: assemblies/assembly-dashbuilder.html [hunk 1]
```diff
@@ -0,0 +1,501 @@
+<!DOCTYPE html>
```
Comment (csherrar) [inline assemblies/assembly-dashbuilder.html]: Is adding `assembly-dashbuilder.html` to the `assemblies` folder intentional? I may be mistaken but is this perhaps a fragment from a doc build that wasn't git-ignored?

File: assemblies/assembly-dashbuilder.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//It looks like heat maps apply to Dashbuilder. Is that right?
+
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-external-components-con.adoc[leveloffset=+2]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-creating-external-components-proc.adoc[leveloffset=+3]
+//It is unclear if this is actually a Dashbuilder component or feature.
+
+== OpenShift Operator
+//[id='assembly-openshift-operator']
+include::{enterprise-dir}/openshift/con-dashbuilder-standalone.adoc[leveloffset=+2]
+include::{enterprise-dir}/openshift/ref-custom-resource-parameters.adoc[leveloffset=+2]
```
Comment (csherrar) [inline assemblies/assembly-dashbuilder.adoc:39]: [style] I think `ref-custom-resource-parameters.adoc` should be offset +3.

File: assemblies/assembly-dashbuilder.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+include::{enterprise-dir}/openshift/proc-dashbuilder-standalone.adoc[leveloffset=+2]
```
Comment (csherrar) [inline assemblies/assembly-dashbuilder.adoc:40]: [style] I think `proc-dashbuilder-standalone.adoc` should be offset +3.

**Review by csherrar — COMMENTED** [review]
Question and two suggestions regarding offset of two sections. Otherwise looks good to me.

File: assemblies/assembly-dashbuilder.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+include::{jbpm-dir}/BAM/Deploying-dashbuilder-data-proc.adoc[leveloffset=+2]
+
+== Building Custom Dashboard Widgets
+//assembly-building-custom-dashboard-widgets
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-heatmap-components-con.adoc[leveloffset=+2]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-creating-process-heatmap-components-proc.adoc[leveloffset=+3]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-creating-all-processes-heatmap-components-proc.adoc[leveloffset=+3]
+include::{enterprise-dir}/pages/building-custom-dashboard-widgets-executing-heatmap-component-proc.adoc[leveloffset=+3]
+]
+//It looks like heat maps apply to Dashbuilder. Is that right?
```
Comment (csherrar) [inline assemblies/assembly-dashbuilder.adoc:30]: @emmurphy1 I believe it applies to Dashbuilder. @mramendi Is this correct and does it need to be retained in assembly-building-custom-dashboard-widgets.adoc ?

File: assemblies/assembly-dashbuilder.adoc (hunk 2 above)
Comment (csherrar) [reply assemblies/assembly-dashbuilder.adoc:39]: This section does not appear in the built preview.

File: assemblies/assembly-dashbuilder.adoc (hunk 3 above)
Comment (csherrar) [inline assemblies/assembly-dashbuilder.adoc:40]: This section does not appear in the built preview.

File: assemblies/assembly-dashbuilder.html (hunk 1 above)
Comment (emmurphy1) [reply assemblies/assembly-dashbuilder.html]: No, just forgot to delete. Thanks @csherrar!

**Pull request merged**